gateway-service-impl/src/main/java/org/hypertrace/gateway/service/explore/RequestHandler.java

```diff
@@ -114,6 +141,92 @@ QueryRequest buildQueryRequest(
     return builder.build();
   }
 
+  private Optional<List<String>> getEntityIdsToFilter(
```

Comment:
not against optional list, but do you need a differentiation  b/w empty optional v/s empty list?

Reply:
missed this. this is not needed